src/tests/conformance/cmd/cluster.rs

```diff
@@ -0,0 +1,63 @@
+use std::net::{IpAddr, SocketAddr, SocketAddrV4};
+
+use tempfile::TempDir;
+
+use crate::{
+    protocol::{
+        codecs::binary::{BinaryMessage, Payload},
+        proto::{TmCluster, TmClusterNode},
+    },
+    setup::{
+        constants::{DEFAULT_PORT, SYNTHETIC_NODE_PUBLIC_KEY},
+        node::{Node, NodeType},
+    },
+    tools::{config::TestConfig, synth_node::SyntheticNode},
+};
+
+#[allow(non_snake_case)]
+#[tokio::test]
+async fn c024_TM_CLUSTER_node_should_connect_to_other_nodes_in_cluster() {
+    // ZG-CONFORMANCE-024
+
+    // SyntheticNode listening ip. Needed for rippled config.
+    let synth_node_ip = "127.0.0.2".parse().unwrap();
+
+    // Start a rippled node with enabled clustering.
+    let target = TempDir::new().expect("unable to create TempDir");
+    let mut node = Node::builder()
+        .enable_cluster(true)
+        .initial_peers(vec![SocketAddr::V4(SocketAddrV4::new(
+            synth_node_ip,
+            DEFAULT_PORT,
+        ))])
+        .start(target.path(), NodeType::Stateless)
+        .await
+        .expect("unable to start rippled node");
+
+    // Start a synthetic node configured to use known keys so that rippled knows who it's talking to.
+    let mut test_config = TestConfig::default();
+    test_config.pea2pea_config.listener_ip = Some(IpAddr::V4(synth_node_ip));
+    test_config.pea2pea_config.desired_listening_port = Some(DEFAULT_PORT);
+    test_config.synth_node_config.generate_new_keys = false;
+    let mut synth_node = SyntheticNode::new(&test_config).await;
```

Comment:
If the node connects to our synth node, should we start the synthetic node before starting the node?
So that our synth node is up before the node tries to connect 🕵️‍♀️

Reply:
It looks like it doesn't matter, but sure, done.